web/frontend/src/reportWebVitals.ts

```diff
@@ -1,15 +0,0 @@
-import { ReportHandler } from 'web-vitals';
```

Comment:
you've imported this dependency in `package.json` . Should we remove the dependency or keep this file ? 
It seems like a nice thing to have https://web.dev/vitals/